### PR TITLE
Updates to code to pass pylint newest version.

### PR DESCRIPTION
**Pull request by Jeff-Meadows**

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @Jeff-Meadows has signed the CLA. Thanks for the pull request!

File: boxsdk/util/log.py [hunk 1]
```diff
@@ -31,9 +31,8 @@ def setup_logging(stream_or_file=None, debug=False, name=None):
     """
     logger = logging.getLogger(name)
     if isinstance(stream_or_file, string_types):
-        handler = logging.FileHandler(stream_or_file, mode='w')
+        logger.addHandler(logging.FileHandler(stream_or_file, mode='w'))
     else:
-        handler = logging.StreamHandler(stream_or_file or sys.stdout)
-    logger.addHandler(handler)
```
Comment (jmoldow) [inline boxsdk/util/log.py]: What did pylint not like about this?

File: test/functional/conftest.py [hunk 2]
```diff
@@ -1,12 +1,14 @@
 # coding: utf-8
 
 from __future__ import unicode_literals
+
+import re
```
Comment (jmoldow) [inline test/functional/conftest.py]: Should be a blank line after this too.

File: test/functional/mock_box/behavior/event_behavior.py [hunk 3]
```diff
@@ -1,10 +1,12 @@
 # coding: utf-8
 
 from __future__ import division, unicode_literals
-from bottle import request
+
 from datetime import datetime, timedelta
+from bottle import request
```
Comment (jmoldow) [inline test/functional/mock_box/behavior/event_behavior.py]: Blank line before here.

File: test/functional/test_events.py [hunk 4]
```diff
@@ -1,9 +1,11 @@
 # coding: utf-8
 
 from __future__ import unicode_literals
+
+from threading import Event, Thread
```
Comment (jmoldow) [inline test/functional/test_events.py]: Blank line after this.

**Comment by jmoldow** [comment]
:+1: , except for a few missing blank lines.

File: boxsdk/util/log.py (hunk 1 above)
Comment (Jeff-Meadows) [inline boxsdk/util/log.py]: It complained that FileHandler and StreamHandler aren't the same types. I think that's a pretty bogus lint error, but figured this change was better than a pylint:disable

**Pull request merged**